tests/test_sentry.py

```diff
@@ -164,6 +164,7 @@ def app(environ, sr):
 
     mw = talisker.sentry.get_middleware(app)
     context = mw.client.context
+    context.clear()
```

Comment:
Note: this is related to getting the tests to pass when you have SENTRY_DSN set in the environment, seemed reasonable to include it.